lib/didomi_sdk.dart

```diff
@@ -185,6 +185,19 @@ class DidomiSdk {
   static Future<String> getTranslatedText(String key) async =>
       await _channel.invokeMethod("getTranslatedText", {"key": key});
 
+  /// Get the user consent status as a UserStatus object
+  static Future<CurrentUserStatus> get currentUserStatus async {
+    final dynamic result = await _channel.invokeMethod("getCurrentUserStatus");
+    return CurrentUserStatus.fromJson(result);
+  }
+
+  /// Set the user consent status as a UserStatus object
```

Comment:
```suggestion
  /// Set the user consent status as a CurrentUserStatus object
```

Reply:
thanks, I just pushed a fix for this function and for the `getCurrentUserStatus`.